css/login-signup.css

```diff
@@ -0,0 +1,92 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  background-color: var(--cyan-300);
+}
+.login-div {
+  margin: 5.6rem auto;
+  height: 36.25rem;
+  width: 28rem;
+  background-color: var(--white);
+  padding: 2rem 3.75rem 1rem;
+}
+.logo {
+  text-align: center;
+  width: 7rem;
+  height: 7rem;
+  border-radius: 100%;
+  margin: 0 auto;
+  margin-bottom: 1.25rem;
+}
+.brand-logo {
+  width: 100%;
+  border-radius: 100%;
+}
+.title,
+.sub-title {
+  text-align: center;
+}
+.title {
+  font-weight: 600;
+  font-size: 2rem;
+}
+.sub-title {
+  font-size: 1.2rem;
+}
+.form {
+  width: 100%;
+  margin-top: 1.87rem;
+}
+.form input {
+  font-size: 1.12rem;
+  padding: 10px 20px 10px 5px;
+  border: none;
+  outline: none;
+  background: none;
+}
+.username,
+.password {
+  display: block;
+  border-radius: 30px;
+  border: 1px solid var(--dark-grey);
+  padding: 10px;
+  margin: 8px 0;
+}
+.icons {
+  padding: 2px;
+  color: var(--dark-grey);
+}
+.options {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 5px 0;
+}
+button.signin-btn {
```

Comment:
- Any Specific reason to use `button.signin-btn` selector here? Why not just `signin-btn` ?

- `signin-btn` class is used for both SignIn and SignUp buttons. class name should be more generic in this case, right now it looks like it's only for SignIn Button

Reply:
Hey I have made the changes suggested by you, please do have a look...

Reply:
Good to merge.